escputil.c:350:15: error: format string is not a string literal (potentially insecure) [-Werror,-Wformat-security]
              printf(_("You may only specify one printer or raw device."));
                     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
../../include/gutenprint/gutenprint-intl-internal.h:58:21: note: expanded from macro '_'
#  define _(String) dgettext (PACKAGE, String)
                    ^~~~~~~~~~~~~~~~~~~~~~~~~~
escputil.c:350:15: note: treat the string as an argument to avoid this
              printf(_("You may only specify one printer or raw device."));
                     ^
                     "%s", 
../../include/gutenprint/gutenprint-intl-internal.h:58:21: note: expanded from macro '_'
#  define _(String) dgettext (PACKAGE, String)
                    ^

Index: src/escputil/escputil.c
--- src/escputil/escputil.c.orig
+++ src/escputil/escputil.c
@@ -22,6 +22,7 @@
 #include <config.h>
 #endif
 #include <gutenprint/gutenprint.h>
+#include <gutenprint/gutenprint-intl-internal.h>
 #include <stdio.h>
 #include <unistd.h>
 #include <string.h>
@@ -52,7 +53,6 @@
 #ifdef HAVE_READLINE_HISTORY_H
 #include <readline/history.h>
 #endif
-#include <gutenprint/gutenprint-intl-internal.h>
 #include "d4lib.h"
 
 void do_align(void);
